vision/src/faker.py

```diff
@@ -8,85 +8,82 @@
 from aquadrone_msgs.msg import Vision_Array, Vision
 
 
+class Omniscient_Vision:
```

Comment:
This file had a mix of tabs and spaces which was fine in python 2, but crashes python 3. There should be no code changes here, only whitespace edits